src/reference/sectionB_elements.inc

```diff
@@ -288,23 +290,23 @@ A :code:`reset` element information item (referred to in this specification as a
    .. container:: issue-reset-variable-reference
 
       1. The :code:`variable` attribute.
-         The value of the :code:`variable` attribute MUST be a valid variable reference, as defined in :numref:`{number}. {name}<specC_variable_reference>`.
+         The value of the :code:`variable` attribute MUST be a valid variable reference, as defined in :numref:`{number} {name}<specC_variable_reference>`.
 
    .. container:: issue-reset-test-variable-reference
 
       2. The :code:`test_variable` attribute.
-         The value of the :code:`test_variable` attribute MUST be a valid variable reference, as defined in :numref:`{number}. {name}<specC_variable_reference>`.
+         The value of the :code:`test_variable` attribute MUST be a valid variable reference, as defined in :numref:`{number} {name}<specC_variable_reference>`.
 
    .. container:: issue-reset-order
 
       3. The :code:`order` attribute.
          The value of the :code:`order` attribute MUST be an :ref:`integer string<specA_integer>`.
 
-         The value of the :code:`order` attribute MUST be unique for all :code:`reset` elements with :code:`variable` attributes that reference variables in the same equivalent variable set (see :numref:`{number}. {name}<specC_interpretation_of_map_variables>`).
+         The value of the :code:`order` attribute MUST be unique for all :code:`reset` elements with :code:`variable` attributes that reference variables in the same equivalent variable set (see :numref:`{number} {name}<specC_interpretation_of_map_variables>`).
```

Comment:
In the headings, we have e.g. `2. Element information items`, so always a `. ` after the number

Reply:
But never after the second number.  We don't refer to chapters, only to sections, which don't have the ending fullstop? `2.1 Something here`

Reply:
Not in the PDF form unfortunately

Reply:
2.1. The model element

Reply:
Which is different to the HTML form

Reply:
2.7. The component element

Reply:
Ahaaaa. It's a PDF / HTML difference

Reply:
PDF wins